src/openrct2/park/ParkFile.h

```diff
@@ -8,7 +8,7 @@ struct ObjectRepositoryItem;
 namespace OpenRCT2
 {
     // Current version that is saved.
-    constexpr uint32_t PARK_FILE_CURRENT_VERSION = 0x9;
+    constexpr uint32_t PARK_FILE_CURRENT_VERSION = 0x10;
```

Comment:
0xA?

Reply:
Why is this hexadecimal in the first place, makes more sense to have this human readable?

Reply:
I assumed that it's so that you can view a (corrupt) park file in a hex editor and immediately see if the version matches, though a decent hex viewer can display numeric values for the selection.

Reply:
I think if you are already using a Hex Editor you have probably enough knowledge to figure that part out as well, a version number should be a decimal value, at least in the source files.